sponsorship/sponsorship-db-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
```

Comment:
All manifests should be stored in dedicated directory or repository, not in service repository or directory

Reply:
Moved to a dedicated directory. It will be moved to a dedicated repository in a separate pull request